test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -80,6 +80,14 @@ export default class BasePageObject {
     return this.getTextContent('[data-se="identifier"]');
   }
 
+  getIdentifierTitle() {
+    return Selector('[data-se="identifier"]').getAttribute('title');
+  }
+
+  identifierHasContenteditable() {
+    return Selector('[data-se="identifier"]').hasAttribute('contenteditable');
+  }
+
```

Comment:
Could you move these to a more specific PageObject? `BasePageObject` is pretty broad.

Reply:
Sure, I can move it into `ChallengePasswordPageObject`. The only reason why I put it here was because the `getIdentifier` method was here, so I wanted it to be grouped together.